### DISCUSSION TITLE
Query regarding the submission of hw8 (Django)

### DISCUSSION THREAD

**Discussion by chintan-wadhwa** [cafawo/FinancialDataAnalytics#24] [Q&A] [answered]
I was able to set up a Django server for the Study Snips project and the server is up and running with the Deribit PTC chart showing on the chart link. However, I'm not sure if the hw is submitted properly since even though I can see the StudySnips folder in my codespace with all the correct datafiles in it (urls.py, views.py etc), the project folder isn't showing up in my repository. Is it supposed to be like that?
Thanks!

**Comment by cafawo** [comment, marked as answer]
Hi @chintan-wadhwa,

Only stuff in your repo counts (I cannot see into your Codespace). Therefore, it makes sense to add your Django project to a subfolder, e.g. "homework/django/".  By the way, you can remove the .git folder from the origin StudySnips repository, as we do not need a link to that repository.

Best!
